pkgs/tools/misc/starship/default.nix

```diff
@@ -22,7 +22,7 @@ rustPlatform.buildRustPackage rec {
   # Delete this on next update; see #79975 for details
   legacyCargoFetcher = true;
```

Comment:
would you mind removing this variable? this would change `cargoSha256`

Reply:
done